packages/analytics/analytics-chart/src/composables/useLineChartOptions.ts

```diff
@@ -124,7 +124,7 @@ export default function useLinechartOptions(chartOptions: LineChartOptions) {
       },
       elements: {
         point: {
-          radius: 0,
+          radius: chartOptions.pointsWithoutHover?.value ? 2 : 0,
```

Comment:
I wonder if the radius should be a bit larger for a single point. Testing it out, I find the "single point" hard to see in the chart

Reply:
per our discussion, how's 3?

our reasoning was: 2 looked small. 4 is the on-hover size and we'd still like an effect, so that leaves 3.